gstreamer-rtsp-server/src/auto/r_t_s_p_client.rs

```diff
@@ -0,0 +1,442 @@
+
+    fn set_session_pool<'a, P: Into<Option<&'a RTSPSessionPool>>>(&self, pool: P);
+
+    fn set_thread_pool<'a, P: Into<Option<&'a RTSPThreadPool>>>(&self, pool: P);
+
+    fn get_property_drop_backlog(&self) -> bool;
+
+    fn set_property_drop_backlog(&self, drop_backlog: bool);
+
+    //fn connect_announce_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //fn connect_check_requirements<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    fn connect_closed<F: Fn(&Self) + Send + Sync + 'static>(&self, f: F) -> SignalHandlerId;
+
+    //fn connect_describe_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //fn connect_get_parameter_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //fn connect_handle_response<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    fn connect_new_session<F: Fn(&Self, &RTSPSession) + Send + Sync + 'static>(&self, f: F) -> SignalHandlerId;
+
+    //fn connect_options_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //fn connect_pause_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //fn connect_play_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_announce_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_describe_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_get_parameter_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_options_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_pause_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_play_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_record_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_set_parameter_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_setup_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn connect_pre_teardown_request<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId;
```

Comment:
Do we have a plan for the RTSPContext btw? Did you try manual bindings and adding it to the manual section as a ref or ref-mut type (similar to caps)? Or does it need even more manual work than that?